src/org/labkey/test/components/ui/grids/EditableGrid.java

```diff
@@ -998,7 +1004,9 @@ protected class ElementCache extends Component<?>.ElementCache
 
         final WebElement table = Locator.byClass("table-cellular").findWhenNeeded(this);
 
-        private final WebElement selectColumn = Locator.xpath("//th/input[@type='checkbox']").findWhenNeeded(table);
+        private final Locator selectAllHeaderCellLoc = Locator.tagWithClass("th", "grid-header-cell")
+            .withAttribute("id", "__selection__");
```

Comment:
A future editor of this file might appreciate a comment indicating why the select-all cell and select-all checkbox are defined separately.
```suggestion
        // Some grids have an placeholder cell without a select-all checkbox
        private final Locator selectAllHeaderCellLoc = Locator.tagWithClass("th", "grid-header-cell")
            .withAttribute("id", "__selection__");
```